pkg/provider/pulumi/gcp/policy.go

```diff
@@ -56,6 +57,12 @@ type PolicyArgs struct {
 	ProjectID pulumi.StringInput
 }
 
+var gcpListActions []string = []string{
+	"pubsub.topics.list",
+	"pubsub.snapshots.list",
+	"resourcemanager.projects.get",
+	"secretmanager.secrets.list",
+}
```

Comment:
@asalkeld has this been tested? I'm wondering if there are any extra permissions we might need to getting resource labels?

Reply:
yip I tested against the test app: https://github.com/asalkeld/test-app
and it passes the tests.. (including the new secrets test)